tools/scripts/platform/linux/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.2.0", 
```

Comment:
did you try without this line ?

Reply:
I belive the version is used to specify what set of arguments will be used inside the json files. So probably you need this line too.

Reply:
I didn't try

Reply:
I get a warning with version missing if I don't add it